package/yast2-storage.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-storage
-Version:        3.2.16
+Version:        3.2.17
```

Comment:
Should it not be 3.2.16.1 since we don't have the fix for bsc#1049108 in master, which is already 3.2.17?